### PR TITLE
[New] Add videojs-thumbnails@0.1.1 w/ git auto-update

### LINKED ISSUE #9821: [Request] Add videojs-thumbnails
**Library name:** videojs-thumbnails
**Git repository url:** https://github.com/brightcove/videojs-thumbnails
License: Apache-2.0

For beginners who is interested in picking up this issue:

In this issue, we are going to host a new web front-end library on CDNJS, so that web developer can easily use it on our free CDN without additional download/upload process 🚀 .

If you'll like to work locally, I'll suggest you start pulling cdnjs repository with sparseCheckout: https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md, since there are so many files in this repo, without sparseCheckout, the whole process will be slow and will use more than 80GB disk space, using git command line can learn more skills and experience, if you don't like that, instead, you can also just do it on GitHub, it's more convenient, but it may be a little bit harder to fix problems on GitHub gui.

Here are the docs about how to add a library:
 - https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#d-adding-a-new-library-with-its-assets
 - https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#e-adding-a-new-library-by-a-single-packagejson

We'll also ask to setup auto-update config so that the added libraries can be updated to the latest version easily:
 - https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md

For beginners who is interested in join and help add this library, please feel free to tell me what kind of help you'll need, we'll help you send the valid pull request, thanks 😄

### PR DESCRIPTION
**Pull request by sashberd**
close #9821, cc @brightcove, using web-helper-minifier for minifiing

Pull request for issue: #9821 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/brightcove/videojs-thumbnails
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference: Apache-2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 28
   - Count of stars: 110
   - Count of forks: 66
 * NPM download stats (optional):
   - Downloads in the last day: 
   - Downloads in the last week: 
   - Downloads in the last month: 

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
Is the npm link official?

**Comment by sashberd** [comment]
@maruilian11 
1) You are right, npm was not of this lib
2) Where I should mention minifer tool and for what? I have problem with webminifier of peterdavehelo, thus I used free online minifiers, there is any problem with it?

**Comment by maruilian11** [comment]
@sashberd Please provide the minify tool link. And mention the tool in the commit msg, like `minify with tool name. `

**Comment by sashberd** [comment]
@maruilian11 the comment was updated

**Comment by maruilian11** [comment]
The commit msg also need to be updated. Thanks.

**Comment by sashberd** [comment]
@maruilian11 The PR was updated

**Comment by maruilian11** [comment]
@sashberd Can you describe how you minify the files? Did you paste it and get minified file online?

**Comment by sashberd** [comment]
@maruilian11 The PR was updated. FYR

**Comment by maruilian11** [comment]
>Did you paste it and get minified file online?

**Comment by sashberd** [comment]
@maruilian11 The PR commit message was updated. FYR

**Comment by sashberd** [comment]
@maruilian11 @extend1994 ping

**Comment by sashberd** [comment]
@extend1994 Please help me to review this PR

File: ajax/libs/videojs-thumbnails/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "videojs-thumbnails",
+  "description": "progress bar thumbnails plugin for video.js",
+  "version": "0.1.1",
+  "author": "Chris Tierney <ctierney@brightcove.com>",
```
Comment (extend1994) [inline ajax/libs/videojs-thumbnails/package.json]: Seems like not the author. He didn't contribute to this project according to https://github.com/brightcove/videojs-thumbnails/graphs/contributors

**Comment by sashberd** [comment]
@extend1994 The main contributor was updated in the author filed

**Comment by sashberd** [comment]
@extend1994 There os now answer from contributors about real author of this lib. May be we ca mention the highest contributor as author in this lib or update this field later

**Comment by extend1994** [comment]
@sashberd What about both `@gkatsev` & `@dmlap`?

**Comment by sashberd** [comment]
@extend1994 I agree with your suggestion. The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 ping

**Pull request merged**